splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/SortOperationIT.java

```diff
@@ -140,6 +146,27 @@ protected void starting(Description description) {
                     ps.setBoolean(1, false);
                     ps.addBatch();
                     ps.executeBatch();
+
+					ps = spliceClassWatcher.prepareStatement("insert into "+ spliceTableWatcher5+" values (?)");
+					ps.setInt(1,1); ps.execute();
+					ps.setInt(1,1); ps.execute();
+					ps.setInt(1,1); ps.execute();
+					ps.setInt(1,5); ps.execute();
+					ps.setInt(1,2); ps.execute();
+					ps.setInt(1,7); ps.execute();
+					ps.setInt(1,4); ps.execute();
```

Comment:
Minor suggestion, feel free to ignore: I'd add a value bigger than some on the other table just to make sure we aren't returning  something like ( ( B order by id) union ( C order by id) )